app/forms/pages/conditions_form.rb

```diff
@@ -36,12 +36,12 @@ def routing_answer_options
     options = page.answer_settings.selection_options.map { |option| OpenStruct.new(value: option.name, label: option.name) }
     options << OpenStruct.new(value: I18n.t("page_options_service.selection_type.none_of_the_above"), label: I18n.t("page_options_service.selection_type.none_of_the_above")) if page.is_optional
 
-    [OpenStruct.new(value: nil, label: I18n.t("helpers.label.pages_conditions_form.default_answer_value")), options].flatten
+    options
   end
 
   def goto_page_options
     page_options = pages_after_current_page(form.pages, page).map { |p| OpenStruct.new(id: p.id, question_text: p.question_with_number) }
-    [OpenStruct.new(id: nil, question_text: I18n.t("helpers.label.pages_conditions_form.default_goto_page_id")), page_options, OpenStruct.new(id: "check_your_answers", question_text: I18n.t("page_conditions.check_your_answers"))].flatten
+    [page_options, OpenStruct.new(id: "check_your_answers", question_text: I18n.t("page_conditions.check_your_answers"))].flatten
```

Comment:
🤔  is `flatten` still needed?

Reply:
Good question! We still need it for adding the check your answers item, but I think we could accomplish the same with either the splat operator:
```suggestion
    [*page_options, OpenStruct.new(id: "check_your_answers", question_text: I18n.t("page_conditions.check_your_answers"))]
```
or by manually adding the item to the array before returning it:
```suggestion
    page_options << OpenStruct.new(id: "check_your_answers", question_text: I18n.t("page_conditions.check_your_answers"))
    
    page_options
```

On reflection I think I prefer both of those to the .flatten approach - have you got any strong opinions on which of those is better?

Reply:
Going to go with the second one for consistency with how we do it in `routing_answer_options` above